### DISCUSSION TITLE
Give option of rearranging files in a PR

### DISCUSSION THREAD

**Discussion by jatzz10** [community/community#10898] [Pull Requests]
I feel that, PR authors should have the ability to rearrange files under `Files Changed` section while creating a pull request.

This will help in - 
- changing the order of files in such a way which will help our reviewers to review much easily, and understand the changes more quickly
- enabling PR authors to structure the PR in a better way

Though this will only be useful in case of long PRs having multiple files.

Thoughts?

**Comment by mernst-github** [comment]
Yes!
